app/templates/report/reports.js.html

```diff
@@ -62,6 +62,11 @@
         }
     });
 
+    var agency_active = "{{ current_user.is_agency_active }}";
```

Comment:
This should only happen (as well as the next 3 lines) if the user is logged in).

If the user is not logged in, then the default should be false (or whatever the equivalent is) since non-agency users cannot view requests sorted by specific agency employee